gax-grpc/src/main/java/com/google/api/gax/grpc/UnaryCallable.java

```diff
@@ -90,160 +63,43 @@
  * // ...
  * ResponseType response = resultFuture.get();
  * }</pre>
+ *
+ * <p>It is considered advanced usage for a user to create a UnaryCallable themselves. This class is
+ * intended to be created by a generated client class, and configured by instances of
+ * UnaryCallSettings.Builder which are exposed through the client settings class.
  */
 @BetaApi
-public final class UnaryCallable<RequestT, ResponseT> {
+public class UnaryCallable<RequestT, ResponseT> implements UnaryCallableImpl<RequestT, ResponseT> {
```

Comment:
Same issue - The class name gives me reverse implication: UnaryCallableImpl implements UnaryCallable